### PR TITLE
test: fix firefox test failures

### PR DESCRIPTION
**Pull request by genehynson**
fixes some cypress failures with firefox specifically now that we're going to be running the cloud tests against firefox

### CODE REVIEW

File: cypress/e2e/cloud/orgs.test.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ describe('Orgs', () => {
     it('forwards the user to the No Orgs Page', () => {
       cy.url().should('contain', 'no-org')
       cy.contains('Sign In').click()
-      cy.url().should('contain', 'dex')
+      cy.location('pathname').should('eq', '/signin')
```
Comment (genehynson) [inline cypress/e2e/cloud/orgs.test.ts:20]: firefox doesn't allow the redirect to dex, so lets just check that we make it to the `/signin` page instead

**Review by hoorayimhelping — APPROVED** [review]

**Comment by hoorayimhelping** [comment]
good find!

File: cypress/e2e/cloud/usage.test.ts [hunk 2]
```diff
@@ -142,7 +142,10 @@ describe('Usage Page PAYG With Data', () => {
     cy.getByTestID('single-stat')
       .should('have.length', 4)
       .each((child, index) => {
-        expect(child.text().trim()).to.equal(stats[index])
+        expect(child.text().trim()).to.be.oneOf([
+          stats[index],
+          Number(stats[index]),
+        ])
```
Comment (genehynson) [inline cypress/e2e/cloud/usage.test.ts:148]: weird issue where firefox doesn't render a comma when the tests are run in circle (but it works fine in real life). So as long as either the string value or numerical value match, we're good to go

File: cypress/support/commands.ts [hunk 3]
```diff
@@ -15,7 +15,8 @@ Cypress.on('uncaught:exception', (err, _) => {
     err.message.includes('Request failed with status code 401') ||
     err.message.includes('The operation was aborted') ||
     err.message.includes('NetworkError') ||
-    err.message.includes('path not found')
+    err.message.includes('path not found') ||
+    err.message.includes('Request aborted')
```
Comment (genehynson) [inline cypress/support/commands.ts:19]: aborted requests should not cause a failure

File: cypress/e2e/cloud/usage.test.ts [hunk 4]
```diff
@@ -142,7 +142,10 @@ describe('Usage Page PAYG With Data', () => {
     cy.getByTestID('single-stat')
       .should('have.length', 4)
       .each((child, index) => {
-        expect(child.text().trim()).to.equal(stats[index])
+        expect(child.text().trim()).to.be.oneOf([
+          stats[index],
+          stats[index].replace(',', ''),
+        ])
```
Comment (genehynson) [inline cypress/e2e/cloud/usage.test.ts:148]: weird issue where firefox doesn't render a comma when the tests are run in circle (but it works fine in real life). So as long as either the string value or numerical value match, we're good to go

**Review by joelfried — APPROVED** [review]
LGTM

**Pull request merged**